setup.py

```diff
@@ -41,31 +41,23 @@ def run(self):
     namespace_packages=['chevah'],
     packages=find_packages('.'),
     package_data={
-        'chevah.github_hooks_server': ['static/*', 'author-aliases.txt'],
+        'chevah.github_hooks_server': ['author-aliases.txt'],
         },
-    scripts=['scripts/start-chevah-github-hooks.py'],
     install_requires=[
-        'klein==17.2',
         'toml',
-        # We keep an older version of python is use.
-        'Twisted==15.5.0.chevah1',
-        'github3-py==1.0.0.gitc82e90e',
-        # We don't depended directly on them but github3.py
-        # dependencies are bad.
-        'urllib3',
-        'chardet',
-        'certifi',
-        'idna',
+        'github3.py',
+        'azure-functions',
+        'azure-functions-worker',
         ],
-    extras_require = {
+    extras_require={
         'dev': [
             'mock',
             'nose',
             'pyflakes',
             'pep8',
             ],
     },
-    test_suite = 'chevah.github_hooks_server.tests',
+    test_suite='chevah.github_hooks_server.tests',
```

Comment:
this was experiment to check testing features in `setup.py`

You can remove it and look at running the tests with `py.test` framework